diff --git a/index.js b/index.js
index 423e20c0d99db50f5a1238674a4420b8bad6c9fb..ed9630ae95840be3e574c234d7571e6939b4ef1f 100644
--- a/index.js
+++ b/index.js
@@ -48,6 +48,7 @@ function versionIncluded(nodeVersion, specifierValue) {
 		: nodeVersion;
 
 	if (typeof current !== 'string') {
+		return true;
 		throw new TypeError(typeof nodeVersion === 'undefined' ? 'Unable to determine current node version' : 'If provided, a valid node version is required');
 	}
 
