diff --git a/src/corePlugins.js b/src/corePlugins.js
--- a/src/corePlugins.js
+++ b/src/corePlugins.js
@@ -1,3 +1,4 @@
-import fs from 'fs'
+// import fs from 'fs'
+import { createFsFromVolume, Volume } from 'memfs'
 import * as path from 'path'
 import postcss from 'postcss'
@@ -566,4 +567,6 @@ export let corePlugins = {
   preflight: ({ addBase }) => {
+    let fs = createFsFromVolume(window.windpress.__MEMFS_VOLUME__ || new Volume())
+
     let preflightStyles = postcss.parse(
       fs.readFileSync(path.join(__dirname, './css/preflight.css'), 'utf8')
     )
